docs/powergate/ffs.md

```diff
@@ -26,11 +45,80 @@ pow ffs create
+
+```bash
+export POW_TOKEN=883f57b1-4e66-47f8-b291-7cf8b10f6370
+```
+
+### Store new data
+
+The FFS requires data you aim to store to be available over IPFS. If you are using the CLI, you can ensure that it is available by staging it on IPFS using `addToHot`. Note that `addToHot` does not store your data in the Powergate FFS. It simply caches your data in the IPFS node in preparation for being stored in the Powergate FFS in the following steps. This is technically equivalent to `ipfs add`, which is adding data without pinning it.
+
+```bash
+pow ffs addToHot <path/filename>
+```
+
+???+ success
+
+    ```Bash
+    Success! Added file to FFS hot storage with cid: <cid>
+    ```
+
+### Push a CidConfig
+
+How the Powergate manages each file stored in the FFS is defined by a _CidConfig_. To tell the Powergate to start managing a new file by moving it from the cached state we created above to the Hot and/or Cold layers, we must push a new CidConfig for the CID we generated above. Learn more about the [CidConfig here](cidconfig.md).
+
+Every FFS instance has a default `CidConfig` that will be used for every new deal unless overridden.
+
+```bash
+pow ffs push --watch <cid>
+```
+
+???+ success
+
+    ```Bash
+    > Success! Pushed cid config for <cid> to FFS with job id: 70368cda-d65a-4e11-8a9f-fbf36135f563
+               JOB ID                	 STATUS    
+    70368cda-d65a-4e11-8a9f-fbf36135f563	Executing
+    ```
+
+When complete, you should see,
+
+???+ success
+
+    ```Bash
+    > Success! Pushed cid config for <cid> to FFS with job id: 70368cda-d65a-4e11-8a9f-fbf36135f563
+               JOB ID                   STATUS
+    70368cda-d65a-4e11-8a9f-fbf36135f563    Success
+    ```
+
+!!!info
+    The FFS Scheduler is configured by default to run up to 50 pushes in parallel, though you can update this setting as needed. [Read more about the FFS design here](https://github.com/textileio/powergate/blob/master/ffs/Design.md).
 
-Powergate provides you API access to a multi-tiered storage system built on IPFS and Filecoin. In many places, we refer to these two tiers of storage as Hot (IFPS) and Cold (Filecoin). Data stored in the Powergate Hot layer is available to the IPFS network (or private network). Data stored only in the Cold layer can be available to the IPFS network, but will require a retrieval deal to pull it from Cold and _add_ it to Hot. This mirrors multi-tiered storage often deployed with a hot storage layer in _memory_ and a cold storage layer on _disk_.
+**FFS Log**
 
-You have a lot of control over how each file is managed on Hot and Cold storage through the use of the [Storage CidConfig](cidconfig.md).
+The status will update as the deal progresses. If you push a file without the `--watch` flag, you can check the logs using `log`.
```

Comment:
Using `log` is to watch for any and all events by cid. If you just want to watch the progress of a specific job, use the `watch <job id>` command